index.rst

```diff
@@ -1,57 +1,435 @@
+    |            | 2020-07 —         | * AuxTel Diagnostic Cluster and Summit Analysis  |
+    |            |                   | * Full DBB from Base to LDF and LDF to Base      |
+    +------------+-------------------+--------------------------------------------------+
+    | ComCam     | 2019-09 — 2019-11 | * Tucson OODS and single-host LSP                |
+    |            |                   | * rsync from Tucson to LDF and Gen2 ingest       |
+    |            +-------------------+--------------------------------------------------+
+    |            | 2019-11 — 2020-01 | * Tucson OODS and single-host LSP                |
+    |            |                   | * Minimal DBB from Tucson to LDF and Gen3 ingest |
+    |            +-------------------+--------------------------------------------------+
+    |            | 2020-01 — 2020-03 | * In transit                                     |
+    |            +-------------------+--------------------------------------------------+
+    |            | 2020-03 — 2020-07 | * Base OODS and Commissioning Cluster LSP        |
+    |            |                   | * Base OODS and Summit Analysis                  |
+    |            |                   | * Minimal DBB from Base to LDF and Gen3 ingest   |
+    |            +-------------------+--------------------------------------------------+
+    |            | 2020-07 —         | * Base OODS and Commissioning Cluster LSP        |
+    |            |                   | * Base OODS and Summit Analysis                  |
+    |            |                   | * Base OODS and OCS-Controlled Batch             |
+    |            |                   | * Full DBB from Base to LDF and LDF to Base      |
+    +------------+-------------------+--------------------------------------------------+
+    | LSSTCam    | 2021-03 —         | * Base OODS and Commissioning Cluster LSP        |
+    |            |                   | * Base OODS and OCS-Controlled Batch             |
+    |            |                   | * Full DBB from Base to LDF and LDF to Base      |
+    +------------+-------------------+--------------------------------------------------+
+
+.. note:: The LSP at NCSA is available at all times for analysis of DBB-conveyed images.
+
+.. _latencies:
+
+Latencies
+=========
+
+The Archivers are expected to transmit images to the OODS and the Data Backbone with a 2-second latency in normal operation.
+The Data Backbone latency is expected to be low in normal operation, but it does more than the OODS in terms of file tracking, and it may experience outages or delays from time to time as it is dependent on more infrastructure services.
+The OODS, on the other hand, is designed to be simpler and have higher uptime and lower latency, so that is the primary immediate-analysis pathway.
+In particular, the "rsync" and "minimal DBB" mechanisms may take more than 15 minutes to begin the transfer of image data.
```

Comment:
Is the “minimal DBB” defined anywhere, beyond being “full DBB with relaxed latency requirements”?

Reply:
I will add some text about the "minimal DBB" to the DBB section.  The DBB design doc is DMTN-122, but it does not yet contain this level of detail.